session 20 part 2 HW/2 table.sql

```diff
@@ -0,0 +1,30 @@
+use CostumeDB
+go
+drop table if exists Costume 
+go 
+create table dbo.Costume(
+    CostumeID int not null identity primary key,
+    CustomerFirstName varchar(30) not null
+        constraint c_Costume_customer_first_name_cannot_be_blank check(CustomerFirstName <> ''),
+    CustomerLastName varchar(30) not null
+        constraint c_Costume_customer_last_name_cannot_be_blank check(CustomerLastName <> ''),
+    CostumeBought varchar(30) not null
+        constraint c_Costume_costumes_bought_cannot_be_blank check(CostumeBought <> ''),
+    Size varchar(2) not null
+        constraint c_Costume_size_must_be_XS_S_M_L_or_XL check(Size in('XS', 'S', 'M', 'L', 'XL')),
+    AmountBought int not null
+        constraint c_Costume_amount_bought_must_be_greater_than_zero check(AmountBought > 0),
+    CostPricePerCostume decimal(4,2) not null
+        constraint c_Costume_cost_price_per_costume_must_be_15_17_20_22_or_25 check(CostPricePerCostume in(15, 17, 20, 22, 25)), 
+    SoldPricePerCostume decimal(4,2) not null,
+    PaidFullPrice bit,
```

Comment:
Don't allow nulls

Reply:
For FullPrice add that if cost and sold price are the same then 0, if not then 1.